src/appMain/core_external_proprietary.sh

```diff
@@ -0,0 +1,127 @@
+#!/bin/bash
+cd $(dirname $0)
+DIR=$(pwd)
+PID_DIR=~/.sdl
+
+if [ ! -d "$PID_DIR" ]; then
+  mkdir $PID_DIR
+fi
+
+CORE_PID_FILE=${PID_DIR}/core.pid
+CORE_PROCESS_NAME=SDLCore
+PM_PID_FILE=${PID_DIR}/policy_manager.pid
+
+function core_start() {
+  if [ -f "$CORE_PID_FILE" ] && [ -n "$(ps -p $(cat $CORE_PID_FILE) -o pid=)" ]; then
+    echo "Core is already running"
+    return 1
+  elif [ -n "$(pgrep $CORE_PROCESS_NAME)" ]; then
+    echo "Core is already running outside of this script"
+    echo "This lingering instance can be stopped using the \"kill\" command"
+    return 2
+  else
+    echo "Starting SmartDeviceLinkCore"
+    LD_LIBRARY_PATH=$DIR ${DIR}/smartDeviceLinkCore &
+    CORE_PID=$!
+    echo $CORE_PID > $CORE_PID_FILE
+    return 0
+  fi
+}
+
+function core_stop() {
+  RESULT=1
+  if [ -f "$CORE_PID_FILE" ] && [ -n "$(ps -p $(cat $CORE_PID_FILE) -o pid=)" ]; then
+    echo "Stopping SmartDeviceLinkCore"
+    CORE_PID=$(cat $CORE_PID_FILE)
+    kill $CORE_PID
+
+    # If Core doesn't close normally within 10 seconds, force-close it
+    timeout 10s tail --pid=$CORE_PID -f /dev/null
+    if [ -n "$(ps -p $CORE_PID -o pid=)" ]; then
+      echo "Core did not shut down properly, force-killing"
+      kill -9 $CORE_PID
+    fi
+    RESULT=0
+  fi
+
+  if [ -f "$CORE_PID_FILE" ]; then
+    rm $CORE_PID_FILE
+  fi 
+  return $RESULT
+}
+
+function pm_install_dependencies() {
+  pip3 list | grep -F tornado > /dev/null
+  if [ $? -eq 1 ]; then
+    echo "Installing tornado python package"
+    sudo pip3 install tornado
```

Comment:
why did we remove -H from sudo? does pip3 not use home dir for installs? it may be default on your install to enable -H but I think we should keep this flag.

Reply:
Seems this was an change carried over from the example in the proposal, which was based on an older version of the script, will fix